grakn-core/src/main/java/ai/grakn/graql/analytics/CentralityQueryBuilder.java

```diff
@@ -37,8 +37,11 @@
      * @param tx the {@link GraknTx} to execute the compute query on
      * @return a compute query builder with the graph set
      */
-    @CheckReturnValue
-    CentralityQueryBuilder withTx(GraknTx tx);
+    //TODO: @Jason. What's the deal with this method?
+    // I remove it and it's implementation and everything still works
+    //@SuppressWarnings("unused")
+    //@CheckReturnValue
+    //CentralityQueryBuilder withTx(GraknTx tx);
 
```

Comment:
The idea was, the user can build a graql query without a tx at the beginning, as long as a tx is provided before execution. So this method is everywhere .

Reply:
@duckofyork is the idea still valid though? I am not sure if we sitll aiming to provide such on the API?